share/revng/pipelines/revng-pipelines.yml

```diff
@@ -9,7 +9,11 @@ Containers:
   - Name: cross-relations.yml
     Type: binary-cross-relations
     Role: cross-relations
-  - Name: module.bc.zstd
+  - Name: functions.bc.zstd
+    Type: llvm-container
+  - Name: root.bc.zstd
+    Type: llvm-container
+  - Name: recompilable-root.bc.zstd
```

Comment:
The name seems to imply that `root.bc.zstd` is not recompilable, while it is, on another branch of the pipeline.

Let's go with `root-and-functions.bc.zstd` or `root-with-functions.bc.zstd`.

Reply:
Done (`with` one).